<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Simulation</string>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="simulation_start_button">
           <property name="text">
            <string>Start</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="simulation_stop_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="simulation_reset_button">
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::SystemReboot"/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Durration</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="simulation_duration_input">
           <property name="suffix">
            <string>s</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Interval</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="simulation_interval_input">
           <property name="suffix">
            <string>ms</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999999</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Połączenie sieciowe</string>
        </property>
        <widget class="QLabel" name="ip">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
        <widget class="QLabel" name="port">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
        <widget class="QLabel" name="Status">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>300</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnPolacz">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>75</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Polacz</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ArxButton">
        <property name="text">
         <string>ARX</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>PID</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Ti</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="pid_ti_input"/>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>      Ti wciągnięte</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Td</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="pid_td_input"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Kp</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="pid_kp_input"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Generator</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Amplitude</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="generator_amplitude_input">
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Period</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="generator_frequency_input">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
           <property name="value">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Infill</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="generator_infill_input">
           <property name="prefix">
            <string/>
           </property>
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Generator tpye</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="generator_generatortype_input">
           <item>
            <property name="text">
             <string>Sine</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Square</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sawtooth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Single Jump</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
        <zorder>generator_infill_input</zorder>
        <zorder>label_3</zorder>
        <zorder>generator_amplitude_input</zorder>
        <zorder>label_11</zorder>
        <zorder>generator_frequency_input</zorder>
        <zorder>label_12</zorder>
        <zorder>generator_generatortype_input</zorder>
        <zorder>label_13</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ChartWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChartWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChartWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_save_as"/>
    <addaction name="action_open"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="action_export"/>
    <addaction name="action_simulation_open"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="action_export">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="action_simulation_open">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartWidget</class>
   <extends>QWidget</extends>
   <header>chartwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
